product-service/src/db.ts

```diff
@@ -0,0 +1,38 @@
+import * as productsList from './mock/products.json';
+
+export interface Product {
+    id: string;
+    imageURL: string;
+    price: string;
+}
+
+class DatabaseServiceMock {
+    products: Map<string, Product>;
+
+    constructor() {
+        this.products = new Map<string, Product>();
+        Array.from(productsList.keys()).forEach(productKey => {
+            const productItem = productsList[productKey];
+            this.products.set(productItem.id, {
+                id: productItem.id,
+                imageURL: productItem.imageURL,
+                price: productItem.price,
+            });
+        });
+    }
+
+    async getProducts() {
```

Comment:
Why? Async without await? Just make function return mocked data like:
function getMockProducts(): Promise<Product[]> {
return Promise.resolve(productsList);
} and getMockProductsById or util function returns Promise.resolve(data);